wrapanapi/systems/virtualcenter.py

```diff
@@ -162,6 +162,41 @@ def host(self):
         self.refresh()
         return self.raw.runtime.host.name
 
+    @staticmethod
```

Comment:
Q: I'm curious why these need to be `staticmethod`s? It's my understanding that a static method can be used similarly to a class method, i.e. it doesn't require an instance of the class to run, and is totally disassociated with the class so doesn't require `self`. So they are basically just external functions? But since they are associated with `VMWareVMOrTemplate` it is more readable to place them with the class?

Reply:
Exactly. These method only pertain to the `VMWareorTemplate` class, so I left them within there.